diff --git a/photon-core/build.gradle b/photon-core/build.gradle
index 2033dbcc..c4de6556 100644
--- a/photon-core/build.gradle
+++ b/photon-core/build.gradle
@@ -18,7 +18,6 @@ def nativeTasks = wpilibTools.createExtractionTasks {
 nativeTasks.addToSourceSetResources(sourceSets.main)
 
 dependencies {
-    wpilibNatives project(path: ':photon-targeting', configuration: 'wpilibNatives')
     wpilibNatives wpilibTools.deps.wpilib("wpimath")
     wpilibNatives wpilibTools.deps.wpilib("wpinet")
     wpilibNatives wpilibTools.deps.wpilib("wpiutil")
@@ -80,9 +79,6 @@ targetingProject.afterEvaluate {
     coreProject.dependencies {
         wpilibNatives files(targetingZipPath)
     }
-    coreProject.tasks.named("processResources") {
-        dependsOn targetingProject.tasks.named("copyAllOutputs")
-    }
 }
 
 task writeCurrentVersion {
